cpp/memilio/math/eigen.h

```diff
@@ -24,6 +24,8 @@
 
 /* this file wraps includes from eigen3 library to disable warnings. */
 
+MSVC_WARNING_DISABLE_PUSH(4996)
```

Comment:
pls provide a comment for future developers

Reply:
done